src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/MigrationService.kt

```diff
@@ -21,51 +23,57 @@ import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.service.Migration
 import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.service.MigrationType
 import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.service.NomisApiService
 
-abstract class PrisonPersonMigrationService(
+@Service("prisonPersonMigrationService")
+class MigrationService(
   queueService: MigrationQueueService,
   private val nomisService: NomisApiService,
-  private val prisonPersonMappingService: PrisonPersonMappingApiService,
+  @Qualifier("prisonPersonMappingApiService") private val mappingApiService: MappingApiService,
   migrationHistoryService: MigrationHistoryService,
   telemetryClient: TelemetryClient,
   auditService: AuditService,
   @Value("\${page.size:1000}") pageSize: Long,
   @Value("\${complete-check.delay-seconds}") completeCheckDelaySeconds: Int,
   @Value("\${complete-check.count}") completeCheckCount: Int,
-) : MigrationService<PrisonPersonMigrationFilter, PrisonPersonMigrationRequest, PrisonerPhysicalAttributesResponse, PrisonPersonMigrationMappingRequest>(
+  private val entityMigrators: List<EntityMigrator<*, *>>,
+) : MigrationService<MigrationFilter, MigrationRequest, PrisonerPhysicalAttributesResponse, PrisonPersonMigrationMappingRequest>(
```

Comment:
The third generic type relates still relates to physical attributes - not sure why this is required, will try and refactor ir out next.